### DISCUSSION TITLE
What is 'your_application_auth_key' ?

### DISCUSSION THREAD

**Discussion by anialamo** [norkunas/onesignal-php-api#164] [General]
$config = new Config('your_application_id', 'your_application_auth_key', 'your_auth_key');

What is the second parameter (your_application_auth_key)? Where should I get it from?

**Comment by norkunas** [comment]
See [#158](https://github.com/norkunas/onesignal-php-api/discussions/158) it's very similar, just look for the `REST API Keys` section
